pkg/exchange/binance/stream.go

```diff
@@ -80,6 +94,12 @@ func NewStream(ex *Exchange, client *binance.Client, futuresClient *futures.Clie
 		futuresClient:  futuresClient,
 		depthBuffers:   make(map[string]*depth.Buffer),
 	}
+	var privatePEM string = os.Getenv("BINANCE_STREAM_PRIVATE_PEM")
```

Comment:
need to use prefix `BINANCE_API_*`, so maybe call it `BINANCE_API_PRIVATE_KEY_PEM`